app/server/services/github.py

```diff
@@ -12,7 +12,11 @@
 
 @lru_cache(maxsize=1)
 class GitHubOperations:
+    """GitHub operations, clone, commit and push"""
```

Comment:
```suggestion
    """GitHub operations, clone, commit and push"""
```

Please add an overview of the class itself.